### ISSUE TITLE
[Fix this bug]Comrademao.com

### ISSUE THREAD

**Issue by damare01** [dipu-bd/lightnovel-crawler#887]
**Describe the bug**
```
An uncaught error was raised while processing the update
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\site-packages\telegram\ext\dispatcher.py", line 279, in process_update
    handler.handle_update(update, self)
  File "C:\Users\user\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\site-packages\telegram\ext\conversationhandler.py", line 305, in handle_update
    new_state = self.current_handler.handle_update(update, dispatcher)
  File "C:\Users\user\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\LocalCache\local-packages\Python38\site-packages\telegram\ext\messagehandler.py", line 169, in handle_update
    return self.callback(dispatcher.bot, update, **optional_args)
  File "C:\Users\user\lightnovel-crawler\.\lncrawl\bots\telegram.py", line 212, in handle_novel_url
    return self.get_novel_info(bot, update, user_data)
  File "C:\Users\user\lightnovel-crawler\.\lncrawl\bots\telegram.py", line 417, in get_novel_info
    app.get_novel_info()
  File "C:\Users\user\lightnovel-crawler\.\lncrawl\core\app.py", line 135, in get_novel_info
    self.crawler.read_novel_info()
  File "C:\Users\user\lightnovel-crawler\.\lncrawl\sources\fu_kemao.py", line 38, in read_novel_info
    self.novel_title = soup.select_one(
```
**Let us know**
- Novel URL: [https://comrademao.com/novel/starry-sky-sovereign/]
- App Version: [v2.26.3]
- Package: [pip]
- OS: [Windows 10]

**Comment by damare01** [comment]
I forgot I opened this issue

**Comment by dipu-bd** [comment]
@damare01 no worry

**Comment by SirGryphin** [comment]
Fixed and added to #929
